.github/pull_request_template.md

```diff
@@ -7,7 +7,7 @@ Fixes # (issue)
 
 ## Type of change
 
-Please delete options that are not relevant.
+Please delete options that aren't relevant.
```

Comment:
for example, is this automated, is it related to the change? Is it its own change. Does the style change remove all contractions? What is this Microsoft folder? Is there a link to what they're acomplishing?